e2e/propExtraction.spec.js

```diff
@@ -63,5 +64,20 @@ describe('Property Extraction in Hierarchies', () => {
         expectedTargetText
       );
     });
+
+    it('properly excludes properties', async () => {
+      // The important thing for this test is that the first mention of 'Button' does NOT include the title property.
+      // This is because the first one is a custom class that specifically excludes (while the second one is the built-in Button.)
```

Comment:
I was confused until I read this. This makes sense. 👍

Reply:
Yeah, since currently the title on Button is the _only_ thing that's possible to exclude (and actually see a difference!)